cluster-up/cluster/external/README.md

```diff
@@ -0,0 +1,36 @@
+# External Kubernetes Provider
+
+This provider works with an existing, provisioned Kubernetes cluster.
+An external Docker registry is recommended for serving images.
+Unlike with other providers, lifecycles of the cluster and registry are not managed.
+The build machine should be a client of the cluster.
+
+## Verifying connectivity
+
+```bash
+export KUBEVIRT_PROVIDER=external
+export DOCKER_PREFIX=myregistry:5000/kubevirt
+export KUBECONFIG=mycluster.conf
+export IMAGE_PULL_POLICY=Always
+make cluster-up
+```
+
+## Building and pushing to the registry
+
+```bash
+make cluster-build
```

Comment:
There's now "cluster-build" directive in the Makefile, this should always fail, and the external provider is now useless because there's no build directive to properly build/push or set the images.

Reply:
actually, we don't have a cluster-build directive in the Makefile, we have an equivalent though, so I can change it so the push happens.

Reply:
> actually, we don't have a cluster-build directive in the Makefile, we have an equivalent though, so I can change it so the push happens.

Actually that's exactly my point s/now/no/  you need to add it; if we're going to copy over the kubevirt directives and model (which I think is a good idea) we should do the whole thing.

Reply:
The point is, the cluster-up is common among the different projects, but each project has different requirements for syncing their stuff into the cluster. The kubevirt model is to put the sync stuff in hack/cluster/something afaict, which is what I did here. hack/cluster/<provider> is where the sync code lives, and cluster-up/<provider> if where the cluster up stuff lives.